train_management/views/traintimetabletemplate.py

```diff
@@ -38,3 +39,29 @@ class TrainTimetableTemplateDeleteView(DvzoDeleteView):
     model = TrainTimetableTemplate
     template_name = "train_management/confirm_delete.html"
     success_url = reverse_lazy("train-timetable-template-list")
+
+
+class CreateTimetablesFromTemplateView(DvzoView):
+    permission_required = 'train_management_add_traintimetable'
+
+    def post(self, request, **kwargs):
```

Comment:
Is there csrf protection implemented using a form like that?

Reply:
When I didn't have the csrf field in the form it didn't work